### PR TITLE
Manually build dev-utils artifact

### PR DESCRIPTION
**Pull request by rsanchez15**
Workflow run can be checked [here](https://github.com/eProsima/eProsima-CI/actions/runs/6664580224)

### CODE REVIEW

File: .github/workflows/build_dev_utils.yml [hunk 1]
```diff
@@ -59,12 +126,28 @@ env:
...
   default_artifact_postfix: _nightly
   artifact_prefix: built_dev_utils
+  default_dev_utils_branch: main
 
 jobs:
 
+  build_fastdds:
+
+    name: build_fastdds
+    uses: eProsima/eProsima-CI/.github/workflows/build_fastdds.yml@feature/manual-devutils-build
```
Comment (JesusPoderoso) [inline .github/workflows/build_dev_utils.yml:136]: I guess that this should be updated to `main` right before merging, isn't it?
If that is true, please apply to all remain `@feature/manual-devutils-build` in both workflows and actions.
```suggestion
    uses: eProsima/eProsima-CI/.github/workflows/build_fastdds.yml@main
```

File: .github/workflows/build_dev_utils.yml [hunk 2]
```diff
@@ -106,8 +178,16 @@ jobs:
         with:
           cmake_build_type: ${{ matrix.cmake_build_type }}
 
-      - name: Get fastdds artifact
+      - name: Get Fast DDS artifact
```
Comment (JesusPoderoso) [inline .github/workflows/build_dev_utils.yml:181]:
```suggestion
      - name: Get Fast DDS built artifact
```

File: .github/workflows/build_dev_utils.yml [hunk 3]
```diff
@@ -106,8 +178,16 @@ jobs:
...
+        uses: eProsima/eProsima-CI/external/download-artifact@feature/manual-devutils-build
+        if: inputs.build_fastdds
+        with:
+          name: built_fastdds_${{ matrix.os }}_${{ matrix.cmake_build_type }}${{ inputs.artifacts_name_postfix || env.default_artifact_postfix }}
+          path: install
+
+      - name: Get Fast DDS artifact
```
Comment (JesusPoderoso) [inline .github/workflows/build_dev_utils.yml:188]:
```suggestion
      - name: Get Fast DDS nightly artifact
```

File: .github/workflows/build_dev_utils.yml (hunk 2 above)
Comment (rsanchez15) [reply .github/workflows/build_dev_utils.yml:181]: Not really. You can run the workflow taking the Fast DDS artifact generated from another manual run. This comment also applies to next review comment.

File: .github/workflows/build_dev_utils.yml (hunk 1 above)
Comment (rsanchez15) [reply .github/workflows/build_dev_utils.yml:136]: True. Do not resolve the comment until the change is applied, which will be done right before merging

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**